### PR TITLE
[WIP] refactor: move to pull-streams

### PR DESCRIPTION
**Pull request by dignifiedquire**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -113,17 +113,25 @@ var node = new ipfsMDAG.DAGNode([<data>, <[links]>])
...
 
-##### getRecursive
+##### getStream
+
+> fetches a node as a pull-stream
+
+##### getRecursiveStream
 
-> fetches a node and all of its links (if possible)
+> fetches a node and all of its links (if possible) as pull-stream
```
Comment (daviddias) [inline README.md]: - [x] add the getRecursive method (without stream)

Comment (dignifiedquire) [reply README.md]: why?

Comment (daviddias) [reply README.md]: good to have both :)

Comment (dignifiedquire) [reply README.md]: but we are not using even one currently :/ and I don't like increasing the api surface

Comment (dignifiedquire) [reply README.md]: well I added it :)

**Comment by dignifiedquire** [comment]
@diasdavid ready

**Pull request merged**